src/UIKit/UICellAccessory.cs

```diff
@@ -0,0 +1,114 @@
+//	Alex Soto <alexsoto@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using ObjCRuntime;
+
+#nullable enable
+#if !WATCH
+
+namespace UIKit {
+
+	public partial class UICellAccessory {
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
+		[DllImport (Constants.UIKitLibrary)]
+		static extern IntPtr UICellAccessoryPositionBeforeAccessoryOfClass (IntPtr accessoryCls);
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
+		[return: DelegateProxy (typeof (SDUICellAccessoryPosition))]
+		[BindingImpl (BindingImplOptions.Optimizable)]
+		public static UICellAccessoryPosition GetPositionBeforeAccessory (Class accessoryClass)
+		{
+			if (accessoryClass == null)
+				throw new ArgumentNullException ("accessoryClass");
+			var ret = UICellAccessoryPositionBeforeAccessoryOfClass (accessoryClass.Handle);
+			return NIDUICellAccessoryPosition.Create (ret)!;
+		}
+		
+		[Watch (7,0), TV (14,0), iOS (14,0)]
+		[return: DelegateProxy (typeof (SDUICellAccessoryPosition))]
+		[BindingImpl (BindingImplOptions.Optimizable)]
+		public static UICellAccessoryPosition GetPositionBeforeAccessory (Type accessoryType) => GetPositionBeforeAccessory (new Class (accessoryType));
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
+		[DllImport (Constants.UIKitLibrary)]
+		static extern IntPtr UICellAccessoryPositionAfterAccessoryOfClass (IntPtr accessoryCls);
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
+		[return: DelegateProxy (typeof (SDUICellAccessoryPosition))]
+		[BindingImpl (BindingImplOptions.Optimizable)]
+		public static UICellAccessoryPosition GetPositionAfterAccessory (Class accessoryClass)
+		{
+			if (accessoryClass == null)
+				throw new ArgumentNullException ("accessoryClass");
+			var ret = UICellAccessoryPositionAfterAccessoryOfClass (accessoryClass.Handle);
+			return NIDUICellAccessoryPosition.Create (ret)!;
+		}
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
+		[return: DelegateProxy (typeof (SDUICellAccessoryPosition))]
+		[BindingImpl (BindingImplOptions.Optimizable)]
+		public static UICellAccessoryPosition GetPositionAfterAccessory (Type accessoryType) => GetPositionAfterAccessory (new Class (accessoryType));
+	} /* class UICellAccessory */
+
+	[UnmanagedFunctionPointerAttribute (CallingConvention.Cdecl)]
+	[UserDelegateType (typeof (UICellAccessoryPosition))]
+	internal delegate nuint DUICellAccessoryPosition (IntPtr block, IntPtr accessories);
+	
+	//
+	// This class bridges native block invocations that call into C#
+	//
+	static internal class SDUICellAccessoryPosition {
+		static internal readonly DUICellAccessoryPosition Handler = Invoke;
+		
+		[MonoPInvokeCallback (typeof (DUICellAccessoryPosition))]
+		static unsafe nuint Invoke (IntPtr block, IntPtr accessories) {
+			var descriptor = (BlockLiteral *) block;
+			var del = (UICellAccessoryPosition) (descriptor->Target);
+			nuint retval = del (NSArray.ArrayFromHandle<UICellAccessory> (accessories));
+			return retval;
+		}
```

Comment:
I believe you can use BlockLiteralRef and avoid it from being unsafe, something like:
```csharp
static unsafe nuint Invoke (ref BlockLiteral block, IntPtr accessories) 
```